packages/create-invoice-form/src/lib/invoice/form.svelte

```diff
@@ -110,6 +110,11 @@
     }
   };
 
+  const handleCheckbox = (event: any) => {
+    formData.isEncrypted = !formData.isEncrypted;
+    setTimeout(() => event.target.checked = formData.isEncrypted, 0);
```

Comment:
Why the the setTimeout instead of something like this for example

```
<Input
    type="checkbox"
    id="isEncrypted"
    label="Encrypt invoice"
    bind:checked={formData.isEncrypted}
/>
```